### DISCUSSION TITLE
New difficulty

### DISCUSSION THREAD

**Discussion by SuperSpinDashGamer39** [KadeArchive/Kade-Engine#1477] [Q&A]
Hey so I need help with making a new difficulty for a mod kinda like alt mode from midfight masses can anyone help mw

**Comment by CallMeInsane** [comment]
in line 9 you should see a array it should have `'Easy', "Normal", "Hard"` add a comma after "Hard" and make some quotation marks
put your new difficulty's name. im not sure about the icon for it i guess you just photoshop it into `assets\preload\images\campaign_menu_UI_assets.png`

edit: i tested and it doesnt work

**Reply by isakube** [reply]
> put your new difficulty's name. im not sure about the icon for it i guess you just photoshop it into `assets\preload\images\campaign_menu_UI_assets.png`
> 
> edit: i tested and it doesnt work

do you know what an xml does by any chance
###### /j

**Reply by Riskerman21** [reply]
thats... a bit to advanced for you so lets just sing about them instead

**Reply by Riskerman21** [reply]
It turns photos into frames is the definition i can think of
